projects/demo/routesFile.txt

```diff
@@ -0,0 +1,28 @@
+/
+/getting-started/what-is-maskito
+/getting-started/maskito-libraries
+/core-concepts/overview
+/core-concepts/mask-expression
```

Comment:
I will create a special utility function to generate this file from [DemoPath](https://github.com/Tinkoff/maskito/blob/main/projects/demo/src/app/demo-path.ts) later in separate PR.

Reply:
* https://github.com/Tinkoff/maskito/issues/240